src/simsopt/_core/optimizable.py

```diff
@@ -1639,7 +1639,6 @@ class TempOptimizable(Optimizable):
         def __init__(self, func, *args, dof_indicators=None, **kwargs):
 
             self.func = func
-            args = np.ravel(args)  # The user may pass a tuple or list
```

Comment:
Why is this deleted?

Reply:
I think that line was present only in the rj/single_stage_PR branch, and not in the master branch. Therefore, its removal makes ml/single_stage_PR even simpler.